client/src/styles/Textbox.module.css

```diff
@@ -0,0 +1,9 @@
+.main {
+    background-color: var(--theme-textbox-background);
+    border: 1px solid var(--theme-border);
+    color: var(--theme-textbox-text);
+    height: 22px;
```

Comment:
Recommend matching the button sizing (`3.6rem` height, `0.4rem` border radius) closer to the Figma design.

Reply:
Done 👍